server/website/website/views.py

```diff
@@ -488,7 +488,7 @@ def handle_result_files(session, files, execution_times=None):
     past_metrics = MetricData.objects.filter(session=session)
     metric_meta = target_objectives.get_instance(session.dbms.pk, session.target_objective)
     if len(past_metrics) > 0:
-        worst_metric = past_metrics.first()
+        worst_metric = past_metrics.order_by('-id').first()
```

Comment:
The value from this line is used when there is no valid metrics at all. If we use the latest metrics, the worst value is doubled every time and expands to a large number. So we need to use the first metric instead.